app/models/mission.rb

```diff
@@ -1,4 +1,16 @@
 # frozen_string_literal: true
 
 class Mission < ApplicationRecord
+  validates :title, presence: true, length: { maximum: 50 }
+  validates :content, presence: true
+  validate :end_after_start_time
+
+  private
+
+  def end_after_start_time
+    error_message = I18n.t('activerecord.errors.models.mission.attributes.end_after_start_time')
+    return if started_at.blank? || ended_at.blank?
+
+    errors.add(:ended_at, error_message) if started_at > ended_at
+  end
```

Comment:
Method 命名請修正。從 `validate :end_after_start_time ` 看不出要 validate 的內容外， method 命名也看不出來是個 custom validator 。建議參考 https://github.com/rubocop/rails-style-guide#custom-validation-methods 或 https://github.com/rubocop/rails-style-guide/issues/223 。

Reply:
- 變數 `error_message` 不需要在 guard clause 前面先指派。到需要加 error 時再指派即可
- 最後一行的 `if started_at > ended_at` 也可以拉到 method 開頭，改以 guard clause 的方式呈現， method 邏輯會更一目暸然
```suggestion
  def end_after_start_time
    return if started_at.blank? || ended_at.blank?
    return if started_at < ended_at
 
    error_message = I18n.t('activerecord.errors.models.mission.attributes.end_after_start_time')
    errors.add(:ended_at, error_message)
  end
```